src/main/scala/gen/gmos/GmosNLongslitD.scala

```diff
@@ -216,31 +350,25 @@ object GmosNLongslitD {
 
       }
 
-      private val emptySequence: Stream[F, Step.GmosN] =
-        Stream.empty.covaryAll[F, Step.GmosN]
-
-      override def sequence(
-        itc:        Itc[F],
-        acquired:   F[Boolean],
-        reachedS2N: F[Boolean]
+      override def sequenceWithReacquisition(
+        itc:             Itc[F],
+        acquired:        F[Boolean],
+        reachedS2N:      F[Boolean],
+        reacquirePeriod: FiniteDuration
       ): Stream[F, Step.GmosN] = {
 
-        // The raw acquisition sequence
-        val acqSequence: Stream[F, Step.GmosN] =
-          acquisition(itc, acquired)
-
         // An infinite Stream[F, Stream[F, Step.GmosN]] where at each step we
-        // check whether an acquisition is necessary and output an empty Stream
-        // if not, or else an acquisition sequence if so.
+        // check whether a reacquisition is necessary
         val reacquireAsNecessary: Stream[F, Stream[F, Step.GmosN]] =
-          Stream.repeatEval {
-            acquired.map { a => if (a) emptySequence else acqSequence }
-          }.cons1(acqSequence) // always do an acquisition sequence at first
+          Stream.every[F](reacquirePeriod).drop(1).map { doReacq =>
+            if (doReacq) reacquisition(itc, acquired) else emptySequence
+          }
```

Comment:
`Stream.every` is supposed to generate a `true` after the given period has elapsed but in fact it also generates `true` immediately.  I find that a bit unexpected but easy to work around by just dropping the first element.

Reply:
```
> println(Stream.every[IO](1 hours).take(2).compile.toVector.unsafeRunSync) 
Vector(true, false)
```